xbmc/guilib/guiinfo/GUIInfoLabels.h

```diff
@@ -643,6 +643,7 @@
 #define PVR_TIMESHIFT_PROGRESS_START_TIME (PVR_STRINGS_START + 70)
 #define PVR_TIMESHIFT_PROGRESS_END_TIME   (PVR_STRINGS_START + 71)
 #define PVR_EPG_EVENT_ICON                (PVR_STRINGS_START + 72)
+#define PVR_TIMESHIFT_SEEKBAR             (PVR_STRINGS_START + 73)
 #define PVR_STRINGS_END                   PVR_EPG_EVENT_ICON
```

Comment:
'#define PVR_STRINGS_END PVR_TIMESHIFT_SEEKBAR' as the new value is now the last string label.

Reply:
Wait, the new label is an int, thus it belongs elsewhere in this file...

Reply:
Ah, now I have it. Those values (for any reason - was before my time) are both labels and ints. So, please add implementation for PVR_TIMESHIFT_SEEKBAR  also to CPVRGUIInfo::GetPVRLabel

Reply:
some are label only, some are int only and some are both. it depends on the usecase.
the label ones can be displayed as text in the skin, the int ones are for use in a progress and/or slider control.
the ones that are a dual use, for instance the Volume infolabel can be shown as text as well a in a volume-bar.

Reply:
Okay, thanks. This PVR_STRINGS_END things needs to be cleanuo up some day (not in this PR). I find it confusing to name items as "strings" if they aren't. OTOH, PVR_STRINGS_END seems not to be used anywhere.